ddev/tests/conftest.py

```diff
@@ -20,6 +24,16 @@ def local_repo() -> Path:
     return Path(__file__).resolve().parent.parent.parent
 
 
+@pytest.fixture(scope='session')
+def valid_integrations(local_repo) -> list[str]:
+    return sorted(path.name for path in local_repo.iterdir() if Integration.is_valid(path))
+
+
+@pytest.fixture
+def valid_integration(valid_integrations) -> str:
+    return secrets.choice(valid_integrations)
```

Comment:
Not a fan of non-deterministic test fixtures.

Reply:
Addressed!

Reply:
Not quite what I meant, `random` is still, well, random 🤷.

Reply:
Oh I see lol. Yeah I was kinda going for this approach https://hypothesis.works

Really I just don't want to pin specific checks

Reply:
Property based testing is quite a different thing, in that case you're running a test through a large amount of random inputs; here you're getting a single random input (a different one for each test).

I think this might be related to some previous discussion we had on a different PR on using a real live repo for tests. I guess we just disagree on this topic, what can I say.